packages/docs/PropTables/InlineAlertPropTable.tsx

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 
-import { PropTable, PropTableWrapper } from '../components';
+import { Code, NextLink, Prop, PropTable, PropTableWrapper } from '../components';
 
 import { sharedMessagingProps } from './shared';
 
+const inlineAlertProps: Prop[] = [
+  ...sharedMessagingProps,
+  {
+    name: 'actions',
+    types: 'object[]',
+    description: (
+      <>
+        Accepts an array of objects with{' '}
+        <NextLink href="/Button/ButtonPage" as="/button">
+          Button
+        </NextLink>{' '}
+        props and an additional <Code>text</Code> prop. See example for usage.
+      </>
+    ),
+  },
+];
```

Comment:
🍹 Can we 🔤 this, by swapping the spread operator?
```suggestion
const inlineAlertProps: Prop[] = [
  {
    name: 'actions',
    types: 'object[]',
    description: (
      <>
        Accepts an array of objects with{' '}
        <NextLink href="/Button/ButtonPage" as="/button">
          Button
        </NextLink>{' '}
        props and an additional <Code>text</Code> prop. See example for usage.
      </>
    ),
  },
  ...sharedMessagingProps,
];
```